packages/react-search-ui/src/containers/Result.js

```diff
@@ -1,18 +1,21 @@
 import PropTypes from "prop-types";
-import React, { Component } from "react";
+import { Component } from "react";
 import { Result } from "@elastic/react-search-ui-views";
 
 import { withSearch } from "..";
 import { Result as ResultType } from "../types";
 
-function getRaw(result, value) {
-  if (!result[value] || !result[value].raw) return;
-  return result[value].raw;
+function getFieldType(result, field, type) {
+  if (!result[field] || !result[field][type]) return;
```

Comment:
What happens if the value you want to get back is `false` or `""` or some other falsey value in JS?

Reply:
It'd return `undefined` I guess.  I'm not opposed to updating this but it was existing code from @JasonStoltz.  Jason, is it good with you if I update to 
```
function getFieldType(result, field, type) {
  if (typeof result[field] == "undefined") return;
  return result[field][type];
}
```

Reply:
I would just return `undefined` here, which I think is OK behavior.

Reply:
My feeling is that if someone has a value of false or an empty string and they ask for "raw" they should get that value.  It may not be a huge issue in practice but I think it is the right behavior.